spk/rutorrent/Makefile

```diff
@@ -1,15 +1,24 @@
 SPK_NAME = rutorrent
 SPK_VERS = 3.10
-SPK_REV = 12
+SPK_REV = 13
 SPK_ICON = src/rutorrent.png
 DSM_UI_DIR = app
 
-SPK_DEPENDS = "python310:PHP5.6"
+SPK_DEPENDS = "WebStation;python310:PHP7.4"
```

Comment:
This syntax may not be correct. Also, having Webstation present may not be all that is needed to change the HTTP back-end server from the default Nginx to Apache HTTP Server 2.4. Any thoughts on getting PHP to work in either Nginx or Apache HTTP Server so that the installation will use whichever the user has already?

Reply:
> This syntax may not be correct.

Indeed, fixed it.

> Also, having WebStation present may not be all that is needed to change the HTTP back-end server from the default Nginx to Apache HTTP Server 2.4. 

No indeed this won't do that. Are we really looking to do this kind of things ?

> Any thoughts on getting PHP to work in either Nginx or Apache HTTP Server so that the installation will use whichever the user has already?

Well there are lots mechanism which could be used but is it really something we want to invest it ? We already need to support lots of different DSMs this may prove to be more complicated than needed (personally I don't care about nginx or apache). To be totally transparent I have limited availabilities so just bear with me if that's the target ;)

Would need to check why apache is needed though.

Reply:
Ideally users should be able to install a package and it work out of the box. Maybe changing the selected server is a config file to set without even needing to install WebStation? If it's onerous to change the selected HTTP back-end server then I would suggest that this requirement be explicit in the wizard UI files so that users will know when installing.

As to why Apache is needed, I couldn't say... curious as to your findings on this.

Reply:
> Ideally users should be able to install a package and it work out of the box. Maybe changing the selected server  is a config file to set without even needing to install WebStation? If it's onerous to change the selected HTTP back-end server then I would suggest that this requirement be explicit in the wizard UI files so that users will know when installing.

I see your point. Not too sure that this is part of this MR's scope though.

> As to why Apache is needed, I couldn't say... curious as to your findings on this.

Would you happen to have logs on the corresponding failures ? Because it may all simply be related to the configuration files not being appropriately injected.

Reply:
so this issue is no longer a problem based on my last comment below...